force_wfmanager/ui/review/plot.py

```diff
@@ -126,6 +115,12 @@ class Plot(HasStrictTraits):
         )
     )
 
+    update_required = Bool(False)
+
+    def _plot_updater_default(self):
```

Comment:
Calling `_update_plot_data` every 1000 ms also calls the `resize_plot` method. Consequently, the graph view is automatically resized after any user change has taken place (such as scrolling / zooming), which should only take place upon firing the Reset View button.

Reply:
I think that's probably okay, since you wouldn't really expect someone to do data analysis things while the workflow is still running. Not resizing the graph view gives the reverse problem, i.e. you zoom in and extra points potentially get added outside of the view

Reply:
Ah, this now resets all the time, even after the workflow manager has completed its BDSS run. So when all the points have been generated the user cannot zoom in at any particular area

Reply:
I will try and make it dependent on the state of the bdss job

Reply:
Refactor: the resize is now triggered only if the data point are changing (usually indicates a job running).

I could add a check button "[  ] Recenter on change of data" but 
1. maybe later?
2. wouldn't it be too much UI for a BasePlot?

Reply:
Great - I agree, nice idea though the current functionality seems fine - I expect we can put these little enhancements in later